editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/internal/l10n/messages.properties

```diff
@@ -218,6 +218,21 @@ updatePomDependenciesProgressDialogLabel=Updating Maven project file for new con
 loadingCamelFile=Loading {0} of project {1}
 pending=Pending ...
 
+editorPreferencePageTechPreviewRESTEditorPageSetting=Enable Tech Preview REST DSL editor (Requires editor restart)
```

Comment:
might be nice to specify that it will be in read-only (can be done in a separated PR)

Reply:
https://issues.jboss.org/browse/FUSETOOLS-2872